src/docsfy/cli/generate.py

```
@@ -141,6 +141,12 @@ def generate(
         None, "--provider", help="AI provider (claude, gemini, cursor)"
     ),
     model: Optional[str] = typer.Option(None, "--model", "-m", help="AI model name"),  # noqa: M511
+    repo_type: Optional[str] = typer.Option(  # noqa: M511
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: M511

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)